appeals/api/src/message-schemas/pins-root.schema.json

```diff
@@ -199,6 +167,78 @@
 					"examples": ["1021477"]
 				}
 			}
+		},
+		"ServiceUser": {
```

Comment:
As these are definitions rather than actual schemas / properties, I thought I capitalise them like classes... Not sure there is a convention but I suggest we keep it. There is already poor convention on some of the properties, but we need to stick to those as defined by ODW.